### DISCUSSION TITLE
Chancomètre

### DISCUSSION THREAD

**Discussion by BananePlantain** [Crownicles/Crownicles#1801] [Suggestions fermées]
J'ai pensé que certains joueurs ont peu de chance en terme d'obtention d'objet et pour cela j'ai réfléchi à un chancomètre.Ce serait une barre de disons 1000 points qui augmente d'un certains nombre quand un objet de rareté commun-spécial est récupéré et se réinitialise (ou pas,ou diminue) pour un objet épique-mythique.Ainsi lors que la barre est à 400,le joueur peut acheter un objet avec les probabilités du Trésor du Roi,500 un épique,700 un légendaire et 1000 un mythique.Les objets communs augmentent la barre de 5,les peu communs de 4...
Les données sont toutes à titres d'exemple et peuvent être modifiées.
Qu'en pensez-vous ?

**Comment by leuhvraiminoryo** [comment]
j'allais faire un long commentaire sur l'équilibrage et comparant ça aux systèmes de pity de certains gachas... finalement flemme

juste une chose : j'suis pour l'idée

**Reply by KirIcare** [reply]
Bro tu lis dans mes pensées, j'allais faire exactement la même ce matin puis nan
Anyways, si ça se fait,je trouverais pertinent une grosse marche entre le légendaire et le mythique (pourquoi pas carrément le double) histoire qu'un véritable dilemme se pose, rester focus ou se "contenter" d'un légendaire ?

**Reply by BananePlantain** [reply]
c'est l'idée de tenter et  de tout perdre

**Comment by symsym-1629** [comment]
J'approuve aussi

**Comment by BastLast** [comment]
Yo, c'est malheureusement impossible à adapter à la manière dont on génère les items. + ca pose pleins d'autres problématiques que j'ai la flemme d'exposer.

**Reply by BananePlantain** [reply]
Dommage,je comprends pour niveau code ca puisse pas suivre même si l'idée avait l'air partagée
